src/spring/azext_spring/_params.py

```diff
@@ -1004,6 +1004,7 @@ def prepare_logs_argument(c):
             c.argument('git_branch', type=str, help='Git repository branch to be used.', validator=validate_acc_git_refs)
             c.argument('git_commit', type=str, help='Git repository commit to be used.', validator=validate_acc_git_refs)
             c.argument('git_tag', type=str, help='Git repository tag to be used.', validator=validate_acc_git_refs)
+            c.argument('git_implementation', type=str, help='Git libraries to be used.', validator=validate_git_implementation)
```

Comment:
Adding `arg_type=get_enum_type` to specify the values parameter could omit the validation of the parameter.
```suggestion
            c.argument('git_implementation', type=str, arg_type=get_enum_type(['go-git', 'libgit2']), help='Git libraries to be used.', validator=validate_git_implementation)
```